docs/experimental-windows.md

```diff
@@ -4,17 +4,13 @@
 
 ## Prerequisites
 
-The cluster must be running at least one worker node and control plane on Linux. You ca use Windows to run additional worker nodes.
+The cluster must be running at least one worker node and control plane on Linux. You can use Windows to run additional worker nodes.
 
-## Build k0s.exe
-
-Invoke the `make clean k0s.exe` command to create k0s.exe with staged kubelet.exe and kube-proxy.exe.
+## Run k0s
 
 **Note**: The k0s.exe supervises kubelet.exe and kube-proxy.exe.
 
-During the first run, the calico install script is creaated as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and set ups vSwitch settings.
-
-## Run k0s
+During the first run, the calico install script is created as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and set ups vSwitch settings.
```

Comment:
"set ups" should probably be "sets up"

Reply:
```suggestion
During the first run, the calico install script is created as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and sets up vSwitch settings.
```

Reply:
Thank you for your suggestion, I updated this.